eclipse-theia/theia
# Add machine translations for 12 languages

#### What it does

Adds support for 12 languages supported by official VSCode language packs. There are some still missing (Turkish, Korean, traditional Chinese), since these aren't translatable by DeepL yet.

Also fixes another round of issues surrounding missing translations. Also makes the `nls-extract` and `nls-translate` commands a bit nicer to work with.

While I'm sure some translations are erroneous or at least sound weird to native speakers, I would like to fix this in separate PRs, if necessary.

#### How to test

1. Install a language pack of your choice (except for the missing ones mentioned above)
2. Use the `Configure Display Language` command to change to that language
3. Inspect the app: Except for extensions, everything should be translated in the app

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
